pkg/status/singletonstatus.go

```diff
@@ -63,25 +63,36 @@ func (c *Controller) updateWorkStatusToObject(ctx context.Context, workStatusON
 
 func (c *Controller) syncWorkloadObject(ctx context.Context, wObjID util.ObjectIdentifier) error {
 	logger := klog.FromContext(ctx)
-	isSingletonRequested, qualifiedWECs := c.bindingPolicyResolver.GetSingletonReportedStateRequestForObject(wObjID)
-	logger.V(4).Info("Workload object (singleton status requested)", "wObjID", wObjID, "isSingletonRequested", isSingletonRequested, "qualifiedWECs", util.K8sSet4Log(qualifiedWECs))
+	isSingletonRequested, qualifiedWECsSingleton, isMultiWECRequested, qualifiedWECsMulti := c.bindingPolicyResolver.GetReportedStateRequestForObject(wObjID)
 
-	// TODO: GetMultiWECReportedStateRequestForObject is yet to be implemented.
-	// just declaring these variables, we will get its actual value once above function is implemented.
-	var isMultiWECRequested bool
-	logger.V(4).Info("Workload object (multiWEC status requested)", "wObjID", wObjID, "isMultiWECRequested", isMultiWECRequested, "qualifiedWECs", util.K8sSet4Log(qualifiedWECs))
+	logger.V(4).Info("Workload object reported state request", "object", wObjID, "isSingletonRequested", isSingletonRequested, "qualifiedWECsSingleton", util.K8sSet4Log(qualifiedWECsSingleton),
+		"isMultiWECRequested", isMultiWECRequested, "qualifiedWECsMulti", util.K8sSet4Log(qualifiedWECsMulti))
 
-	if (isMultiWECRequested || isSingletonRequested) && qualifiedWECs.Len() == 1 {
-		logger.V(4).Info("Either singleton or multiWEC status is requested and qualifiedWECs.Len() == 1 for object", "wObjID", wObjID, "qualifiedWECs", util.K8sSet4Log(qualifiedWECs))
-		return c.handleSingleton(ctx, wObjID, qualifiedWECs)
+	if isMultiWECRequested && isSingletonRequested {
+		logger.V(4).Info("Both singleton and multiWEC reported state return are requested for the same object", wObjID)
```

Comment:
This information already appeared in the previous log statement. Don't junk up the log with repeating it here. This comment also applies to the other redundant log statments below.